content/docs/user-guide/gems/reference/aws/aws-gamelift/flexmatch/cpp-api.md

```diff
@@ -0,0 +1,257 @@
+---
+linkTitle: FlexMatch C++ API
+title: FlexMatch C++ API 
+description: Learn how to use the FlexMatch C++ API with the AWS GameLift Gem in Open 3D Engine (O3DE).
+toc: true
+weight: 300
+---
+
+
+## Client Side
+
+The **AWS GameLift** Gem implements a matchmaking interface for Amazon GameLift. The *matchmaking interface* (`IMatchmakingRequests` and `IMatchmakingAsyncRequests`) provides public APIs to add player matchmaking functionality to your GameLift hosted games. 
+
+The matchmaking interface represents the actions required to handle matchmaking in a game. The Gem provides an implementation to this interface which utilizes GameLift FlexMatch to find and group players in a session for your game.
+
+There must be only one implementation of the matchmaking interface per matchmaking solution. To add support for another matchmaking solution, you must create another implementation of the matchmaking interface.
+
+
+### Matchmaking APIs
+
+#### `StartMatchmaking`
+
+Create a game match for a group of players. Check [StartMatchmaking](https://docs.aws.amazon.com/gamelift/latest/apireference/API_StartMatchmaking.html) for more details.
```

Comment:
I know this is from the GameLift docs but IMHO :game match" is not clear what this is doing to me. Its also not a consistent term wrt to GameLift.

- We are also missing that this goes all the way from find a match -> to starting a game session for that match.

"Find a match for a group of players and create a session to host match"?